MapleServer2/PacketHandlers/Login/CharacterManagementHandler.cs

```diff
@@ -185,7 +185,7 @@ public void HandleCreate(LoginSession session, PacketReader packet)
             AccountStorage.AddCharacter(newCharacter);
 
             // Send updated CHAR_MAX_COUNT
-            session.Send(CharacterListPacket.SetMax(4, 6));
+            session.Send(CharacterListPacket.SetMax(6, 12));
```

Comment:
why this was change? and in others locations too, there is just 3 characters, doesn't need to be change.

Reply:
Oops! Soz. This was a testing thing. Will revert it :)